### DISCUSSION TITLE
Portable Links

### DISCUSSION THREAD

**Discussion by james-d-elliott** [thuliteio/doks#566] [Q&A] [answered]
I was under the impression portable links were possible, however I can't seem to get them to work. If I create a link to a relative file like `[text](file.md)` it doesn't seem to work while running hugo server with `npm run start`. Is there something obvious I'm missing?

**Comment by h-enk** [comment, marked as answer]
No, not yet (but portable image links are).

I will make this an issue to be added soon (next release)!

This is how: https://github.com/bep/portable-hugo-links/blob/master/layouts/_default/_markup/render-link.html

**Reply by james-d-elliott** [reply]
Ah neat! Looks like since we ~~use~~ are experimenting with the forked version for the increased customization we can easily apply that.

**Reply by james-d-elliott** [reply]
Yep you just have to add that file, instant success.
